collator/runtime/src/lib.rs

```diff
@@ -460,6 +784,17 @@ construct_runtime! {
 		PhalaXcmTransactor: xcm_transactor::{Module, Call, Event<T>},
 		Council: pallet_collective::<Instance1>::{Module, Call, Storage, Origin<T>, Event<T>, Config<T>},
 		Treasury: pallet_treasury::{Module, Call, Storage, Config, Event<T>},
+		Identity: pallet_identity::{Module, Call, Storage, Event<T>},
```

Comment:
The additional pallets must be appended at the end of the config. Otherwise it will be incompatible with the older runtime￼￼￼￼